cmd/transcriber/call/live_captions.go

```diff
@@ -0,0 +1,323 @@
+package call
+
+import (
+	"fmt"
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/apis/whisper.cpp"
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/config"
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/transcribe"
+	"github.com/streamer45/silero-vad-go/speech"
+	"log/slog"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+)
+
+const (
+	// TODO: these need to be in cfg and env var settable, and in proper style.
+	parallelTranscribersPool = 4
+	threadsPerTranscriber    = 1
+
+	chunkSizeInMs            = 1000
+	maxWindowSizeInMs        = 10000
+	audioDataChannelBuffer   = 30 // so we don't block while processing the window's vad
+	removeWindowAfterSilence = 3 * time.Second
+
+	// VAD settings
+	vadWindowSizeInSamples  = 512
+	vadThreshold            = 0.5
+	vadMinSilenceDurationMs = 300
+	vadMinSpeechDurationMs  = 200
+	vadSilencePadMs         = 32
+)
+
+// TODO: belongs in calls-common?
+type CaptionMsg struct {
+	SessionID string `json:"session_id"`
+	UserID    string `json:"user_id"`
+	Text      string `json:"text"`
+}
+
+type captionPackage struct {
+	pcm []float32
+	ret chan string
```

Comment:
Can we use the `Ch` suffix to be super explicit on what `ret` is when used elsewhere?

Reply:
Yep, good idea.